waterlog/src/Components/Map.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
+import {sensorFaultIcon, sensorOkIcon} from './../icons/MapIcons';
+
+var markers = [
+	{ lat: -25.782473, lon: 28.338061 },
+	{ lat: -25.782454, lon: 28.337993 },
+	{ lat: -25.782456, lon: 28.337866 },
+    { lat: -25.782634, lon: 28.338039 },
+    { lat: -25.783408, lon: 28.336180}
+];
+
+export default class MapComponent extends Component {
+	constructor() {
+		super();
+		this.state = {
```

Comment:
You might as well initialize your state with the markers array of object at the third position

Reply:
I created the markers array in the state and initialized it